shunit2_test_helpers

```diff
@@ -226,6 +226,13 @@ _th_showOutput() {
   unset _th_return_ _th_stdout_ _th_stderr_
 }
 
+# Constants used to test file and directory assertions
```

Comment:
As these are only used in the one unit test, they should be present in just that file rather than here. It makes it easier for somebody reading the code to find them. Also, it would make the tests cleaner if the constants were defined explicitly only in the test they were used in.